src/main.c

```diff
@@ -281,10 +281,15 @@ __attribute__((section(".boot"))) int main(void) {
           io_seproxyhal_init();
 
           // Consider using an internal storage thingy here
-
           USB_power(0);
           USB_power(1);
 
+          #if defined(TARGET_NANOX)
+          G_io_app.plane_mode = os_setting_get(OS_SETTING_PLANEMODE, NULL, 0);
```

Comment:
What is this doing exactly? are you using it later in the code?

Reply:
This is the boilerplate for BLE support. Haven't really looked what the SDK does under the hood, but I'd imagine that when the user has disabled bluetooth (globally) then it won't initialise the radio (thus the airplane mode name).